engine/src/cmd/basecomputer.cpp

```diff
@@ -527,7 +529,7 @@ void BaseComputer::constructControls(void) {
         return;
     }
 
-    if (m_displayModes.size() != 1 || m_displayModes[0] != NETWORK) {
+    if (m_displayModes.size() != 1 || m_displayModes.at(0) != NETWORK) {
```

Comment:
I assume these fail for some OS'es. I have the dangest errors in the CI, where my machine blissfully compiles and runs.

Reply:
These don't fail to compile on any OS particularly. This is just to add bounds checking, to make sure we're not reading or writing outside the bounds of the collection.